index.js

```diff
@@ -151,7 +151,13 @@ module.exports = {
         });
 
         each(function(entry) {
-          entry.url['xhtml:link'] = [];
+          entry.url['xhtml:link'] = [{
```

Comment:
This should be conditional on Workflow being in place, right? Maybe even just there being `entry.url.workflowLocale`.